includes/widgets/common.php

```diff
@@ -364,6 +364,205 @@ protected function register_controls() {
 
 		$this->end_controls_section();
 
+		$this->start_controls_section(
```

Comment:
This feature will work only on WebKit based browsers. You may add a notice to let the users aware of it.

Reply:
Firefox can handle webkit.